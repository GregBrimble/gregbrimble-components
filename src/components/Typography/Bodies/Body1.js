import React from "react"

import styled from "styled-components"
import { Body } from "./shared"

const Body1 = styled(Body)`
  font-size: 1rem;
  line-height: 1.5rem;
`

export default Body1
