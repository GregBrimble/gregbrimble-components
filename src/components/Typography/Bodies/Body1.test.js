import React from 'react';
import Body1 from './Body1';
import renderer from 'react-test-renderer';

describe('Body1', () => {
    it('renders properly', () => {
        const tree = renderer
            .create(<Body1>Body 1</Body1>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
