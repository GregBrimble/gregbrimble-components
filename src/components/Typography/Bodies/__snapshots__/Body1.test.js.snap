// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body1 renders properly 1`] = `
<p
  className="sc-bdVaJa sc-bwzfXH ejKZWa"
>
  Body 1
</p>
`;
